addons/website_sale/data/website_sale_tour.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="test_01_admin_shop_tour" model="web_tour.tour">
+        <field name="name">test_01_admin_shop_tour</field>
+        <field name="sequence">130</field>
+    </record>
```

Comment:
```suggestion
<odoo>

    <record id="test_01_admin_shop_tour" model="web_tour.tour">
        <field name="name">test_01_admin_shop_tour</field>
        <field name="sequence">130</field>
    </record>

```

Personal opinion, but it reduces conflicts and is more enjoyable to read with spaces :eyes: